src/detectors/EEMC/Cluster_factory_EcalEndcapNTruthClusters.h

```diff
@@ -45,8 +45,8 @@ class Cluster_factory_EcalEndcapNTruthClusters : public JFactoryT<edm4eic::Clust
         m_input_simhit_tag="EcalEndcapNHits";
         m_input_protoclust_tag="EcalEndcapNTruthProtoClusters";
     
-        m_sampFrac=1.0;//{this, "samplingFraction", 1.0};
-        m_logWeightBase=3.6;//{this, "logWeightBase", 3.6};
+        m_sampFrac=0.03;//{this, "samplingFraction", 1.0};
+        m_logWeightBase=4.6;//{this, "logWeightBase", 3.6};
```

Comment:
Just out of curiosity as well here, what is the unit of these 2 values?
what is the difference the m_sampFrac and the sampling fraction we used to correct energy?
Thanks!

Reply:
Sampling fraction is a number between 0 and 1: how much energy is deposited in sensitive elements (as opposed to radiators). For homogeneous approximations of microstructure, this is expected to be different from reality.

The log weight base is a dimensionless constant that ln(E/Etotal) is added to in the center of gravity cluster weighting. A larger values means more uniform weighting; a smaller value means more logarithmic weighting.

Reply:
Very thanks, it is much more clear now. Final question, do you know how to estimate the log weight? By the ratio of center seed energy and rest of hit energy?

Reply:
No idea. Maybe @chao1009 or @mariakzurek knows.

Reply:
Use b = -ln(x), with x = hit_energy/cluster_energy to estimate the base.
For example, b = 4.6 means those hits with energy less than 1% of the cluster energy won't participate in coordinate reconstruction.

Reply:
Thanks for answering!